### PR TITLE
Peter review

### PR DESCRIPTION
**Pull request by merrygoat**
This is a review of lessons 1 to 3.

### CODE REVIEW

File: _episodes/01-introduction.md [hunk 1]
```diff
@@ -40,24 +40,24 @@ We wish to create a list of values, which of these lines of code is valid to do
...
-2. `print(variable[4:])`
-2. `print(variable[4:5])`
-3. `print(variable[3:5])`
-4. `print(variable[-2:])`
-5. `print(variable[:-2])`
+We can also select whole sections of lists and strings, not just single elements. Using the same variable `animal`, containing the string `penguin`, as above, which of these options will print the last 2 characters for us? (Note that there is more an one correct answer)
+1. `print(animal[3:])`
+2. `print(animal[4:])`
+2. `print(animal[4:5])`
+3. `print(animal[3:5])`
```
Comment (douglowe) [inline _episodes/01-introduction.md:58]: This does show why `-2` is the more robust solution ;-)
Can you change to these slices:
1. [5:]
2. [6:]
3. [6:7]
4. [5:7]

File: _episodes/02-dictionaries.md [hunk 2]
```diff
@@ -267,6 +277,28 @@ dict_values([12, 18, 24, 54, 87])
...
+~~~
+Traceback (most recent call last):
+  File "<stdin>", line 1, in <module>
+TypeError: 'dict_keys' object is not subscriptable
+~~~
+{: .output}
+
+If you want to index keys or values directly, you can convert them to lists with the `list` function.
+~~~
+d.values()[0]
```
Comment (douglowe) [inline _episodes/02-dictionaries.md:294]: Could you change this to:
list(d.values())[0]

**Review by douglowe — CHANGES_REQUESTED** [review]
Good additions - thanks. Can you make the copy of changes noted above, and then I'll merge the PR.

File: _episodes/02-dictionaries.md (hunk 2 above)
Comment (merrygoat) [reply _episodes/02-dictionaries.md:294]: I completely failed to make the point I was trying to make with that omission!

**Comment by douglowe** [comment]
Thanks :)

**Pull request merged**